src/index.html

```diff
@@ -11,7 +11,118 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form__list">
+        <legend>Personal information</legend>
+        <label for="sname" class="form__list-item">Surname:
+          <input
+           type="text"
+           id="sname"
+           autocomplete="off">
+        </label>
+        <label for="name"  class="form__list-item">Name:
+          <input
+           type="text"
+           id="name"
+           autocomplete="off">
+        </label>
+        <label for="old" class="form__list-item">How old are You?
+          <input
+           id="old"
+           type="number"
+           value="12"
+           min="10"
+           max="60" >
+        </label>
+        <label for="birth" class="form__list-item">Full date of birth:
+          <input type="date" id="birth">
+        </label>
+        <label id="terms" class="form__list-item">I accept the term of the agreement
+          <input for="terms"  type="checkbox">
+        </label>
+      </fieldset>
+
+      <fieldset class="form__list">
+        <legend >Registration</legend>
+        <label for="email" class="form__list-item">E-mail:
+          <input
+           type="email"
+           id="email"
+           placeholder="email@example.com"
+           autocomplete="off">
+        </label>
+        <label for="pass"  class="form__list-item">Password:
+          <input
+           type="password"
+           minlength="8"
+           maxlength="32"
+           id="pass"
+           autocomplete="off">
+        </label>
+      </fieldset>
+
+      <fieldset class="form__list">
+        <legend>An interesting fact about you!</legend>
+        <label for="cats" class="form__list-item">Do you love cats?
+          <input
+           id="cats"
+           type="radio"
+           name="cats"
+           class="cats"
+           value="yes"
+           >Yes
+          <input
+           class="cats"
+           type="radio"
+           name="cats"
+           value="no">No
+        </label>
+        <label for="color"  class="form__list-item">What is your favorite color?
+          <input type="color" id="color">
+        </label>
+        <label for="time" class="form__list-item">What time do you go to bed?
+          <input
+           id="time"
+           type="time"
+           step="2">
+        </label>
```

Comment:
+ add blank lines between same level elements
```suggestion
          <input
           class="cats"
           type="radio"
           name="cats"
           value="no"
         >
         No
        </label>
        
        <label for="color"  class="form__list-item">
          What is your favorite color?
          <input type="color" id="color">
        </label>
        
        <label for="time" class="form__list-item">
          What time do you go to bed?
          <input
             id="time"
             type="time"
             step="2"
           >
        </label>
```